vpr/src/base/check_netlist.cpp

```diff
@@ -48,20 +48,19 @@ void check_netlist(int verbosity) {
     /* Check that nets fanout and have a driver. */
     int global_to_non_global_connection_count = 0;
     for (auto net_id : cluster_ctx.clb_nlist.nets()) {
-        h_net_ptr = insert_in_hash_table(net_hash_table, cluster_ctx.clb_nlist.net_name(net_id).c_str(), size_t(net_id));
+        h_net_ptr
+            = insert_in_hash_table(net_hash_table, cluster_ctx.clb_nlist.net_name(net_id).c_str(), size_t(net_id));
```

Comment:
BreakBeforeBinaryOperators: All

Reply:
Ugly. Prefer the BreakBeforeBinaryOperators: NonAssignment to keep the = on the first line.

Reply:
I am indifferent on this one. My personal preference for this line would be:
```cpp
h_net_ptr = insert_in_hash_table(net_hash_table, 
                                 cluster_ctx.clb_nlist.net_name(...),
                                 ...)
```
Or the line above should use more temporary variables to make this look nicer (which is good practice on the programmer's part).